code/modules/ghostroles/spawner/human/admin.dm

```diff
@@ -112,4 +117,18 @@
 	respawn_flag = null
 
 	mob_name = null
-	mob_name_prefix = "Spec. "
\ No newline at end of file
+	mob_name_prefix = "Spec. "
+
+
+
+/client/proc/despawn()
+	set name = "Despawn"
+	set desc = "Your work is done. Leave this realm."
+	set category = "Special Verbs"
+
+	var/mob/M = mob
```

Comment:
```suggestion
```

Should be unnecessary.

Reply:
Is necessary so the proper mob can be deleted after we ghosted them.